test/fail_compilation/fail17625.d

```diff
@@ -0,0 +1,17 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail17625.d(16): Deprecation: `b17625.boo` is not visible from module `fail17625`
+fail_compilation/fail17625.d(16): Error: function `b17625.boo` is not accessible from module `fail17625`
+---
+*/
+
+module fail17625;
+
+import imports.a17625;
```

Comment:
FYI: You are importing the modules using `imports.a17625` but then you declare the module name in the file to be `a17625`.  I'm hoping to deprecate this via https://github.com/dlang/dmd/pull/7778